### ISSUE TITLE
Deprecation /containermetrics endpoint on the loggregator_trafficcontroller

### ISSUE THREAD

**Issue by chenl23** [chenl23/cfar-proposals#7]
### Proposal Name
Deprecation /containermetrics endpoint on the loggregator_trafficcontroller

### TL;DR Summary
The loggregator team would like to remove the /containermetrics endpoint on the loggregator_trafficcontroller (see here: https://github.com/cloudfoundry/loggregator-release/blob/master/docs/trafficcontroller.md#endpoints). This endpoint originally was introduced as an endpoint for the Cloud Controller to fetch metrics about the running container.  As of cf-deployment v7.2 (https://github.com/cloudfoundry/cf-deployment/releases/tag/v7.2.0), Cloud Controller now reaches out to log-cache to fetch the container metrics. Therefore, we would like to remove this endpoint to keep our API and code base lean and clean. If there are consumers of this endpoint besides the Cloud Controller, please let us know. 

### Proposal URL
https://lists.cloudfoundry.org/g/cf-dev/message/8481

### Point of Contact
Fred Krone

**Comment by chenl23** [comment]
Removed in [Loggregator release v105](https://github.com/cloudfoundry/loggregator-release/releases)
